src/libraries/System.IO.Compression/tests/ZipArchive/zip_LargeFiles.cs

```diff
@@ -51,8 +51,7 @@ private static void FillWithHardToCompressData(byte[] buffer)
 
     [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMobile), nameof(PlatformDetection.Is64BitProcess))] // don't run it on slower runtimes
     [OuterLoop("It requires 5~6 GB of free disk space and a lot of CPU time for compressed tests")]
-    [InlineData(false)]
-    [InlineData(true)]
+    [MemberData(nameof(Get_Booleans_Data))]
```

Comment:
should this test be also extended with async code path coverage?

Reply:
This is an outerloop test. I replaced the two inline datas (false and true booleans) in bulk, and this one got caught coincidentally.

I thought about this and just to be safe, I created a copy of the same test but in async mode.